src/CFamily.UnitTests/Rules/CFamilySonarWayRulesConfigProviderTests.cs

```diff
@@ -32,9 +32,13 @@ public class CFamilySonarWayRulesConfigProviderTests
         // Rule data for files in Rules\TestResources\RulesMetadataCache
         private const int Active_C_Rules = 3;
         private const int Inactive_C_Rules = 3;
+        private const int Active_Misra_C_Rules = 1;
+        private const int Inactive_Misra_C_Rules = 1;
 
         private const int Active_CPP_Rules = 4;
         private const int Inactive_CPP_Rules = 2;
+        private const int Active_Misra_Cpp_Rules = 1;
+        private const int Inactive_Misra_Cpp_Rules = 1;
```

Comment:
Fixing up the existing tests.
I added a dummy json files for four rules (C/C++, Active/Inactive), and listed them in a dummy `MisraRulesList.json` file.